extensions/gdx-tools/src/com/badlogic/gdx/tools/etc1/ETC1AtlasCompressionResult.java

```diff
@@ -0,0 +1,59 @@
+

```

Comment:
Practically the differences between all open source licenses exist only on paper, while in fact they all are the Unlicense. Anyway the Unlicense allows anyone to use it under the Apache too.